extensions/data-workspace/src/main.ts

```diff
@@ -35,6 +35,9 @@ export function activate(context: vscode.ExtensionContext): Promise<IExtension>
 	context.subscriptions.push(vscode.commands.registerCommand('dataworkspace.refresh', () => {
 		workspaceTreeDataProvider.refresh();
 	}));
+	context.subscriptions.push(vscode.commands.registerCommand('dataworkspace.close', () => {
+		workspaceTreeDataProvider.close();
```

Comment:
it's probably ok to directly call `vscode.commands.executeCommand('workbench.action.closeFolder')` here

Reply:
yeah, this feature isn't really a tree data provider feature, it is a top level feature. directly call the command here.